projects/novo-elements/src/elements/form/NovoFormControl.spec.ts

```diff
@@ -2,7 +2,45 @@
 import { Validators } from '@angular/forms';
 
 // App
-import { NovoFormControl } from './NovoFormControl';
+import { NovoFormControl, NovoFormGroup } from './NovoFormControl';
```

Comment:
Since you're already doing this, it makes sense to put `NovoFormGroup` and the specs in separate files.